types/actors.md

```diff
@@ -30,13 +30,59 @@ Here we have an `Aardvark` that can eat asynchronously. Clever Aardvark.
 
 ## Concurrent
 
-Since behaviours are asynchronous, it's ok to run the body of a bunch of behaviours at the same time. This is exactly what Pony does. The Pony runtime has its own scheduler, which by default has a number of threads equal to the number of CPU cores on your machine. Each scheduler thread can be executing an actor behaviour at any given time, so Pony programs are naturally concurrent.
+Since behaviours are asynchronous, The runtime might execute a bunch of behaviours at the same time. This is exactly what Pony does. The Pony runtime has its own scheduler, which by default has a number of threads equal to the number of CPU cores on your machine. Each scheduler thread can be executing an actor behaviour at any given time, so Pony programs are naturally concurrent.
+
+for example, the following code:
+
+```
+actor A
+  var name:String
+  var env:Env
+  new create(env':Env,name':String) =>
+    name = name'
+    env = env'
+
+  be first() =>
+    env.out.print(name+" first")
+  be second() =>
+    env.out.print(name+" second")
+ 
+actor Main
+
+  new create(env:Env) =>
+     let a = A.create(env,"a")
+     let b = A.create(env,"b")
+     a.first()
+     a.second()
+     b.first()
+     b.second()
+
+```
+might have the following resul:
+
+```
+b first
+a first
+b second
+a second
+```
```

Comment:
Your example result isn't actually possible in Pony, as you point out below. Therefore I worry that this section is misleading and could confuse someone. I don't think there's any need to introduce this level of potential confusion here.